src/pyfaf/utils/contextmanager.py

```diff
@@ -1,5 +1,12 @@
 import sys
-from StringIO import StringIO
+
+try:
+#Python 2
+    from StringIO import StringIO
+except ImportError:
+#Python 3
+    from io import StringIO
+
```

Comment:
It is probably matter of preference, but I prefer:

```
if six.PY3:
  ...
```
Your style can hide issues when it is Python2 and there is some error during import of StringIO (other than module is missing).